### PR TITLE
[C-API/Test] Apply test fixture to remove duplicate code

### PR DESCRIPTION
**Pull request by again4you**
To remove duplicate code in each test case, this patch applies the
test fixture framework. g_autofree macro is used for allocated space
to be freed automatically no matter test case is passed or not.

Signed-off-by: Sangjung Woo <sangjung.woo@samsung.com>

### Self evaluation:
Build test: [X]Passed [ ]Failed [ ]Skipped
Run test: [X]Passed [ ]Failed [ ]Skipped

#### v2
* Update the code to comply with coding style.

#### v1
* Apply test fixture framework of Google Test

### CODE REVIEW

File: tests/capi/unittest_capi_service.cc [hunk 1]
```diff
@@ -12,143 +12,102 @@
...
-  /* supposed to run test in build directory */
-  if (root_path == NULL)
-    root_path = "..";
-
-  test_model =
-      g_build_filename (root_path, "tests", "test_models", "models",
-      "add.tflite", NULL);
-  EXPECT_TRUE (g_file_test (test_model, G_FILE_TEST_EXISTS));
+protected:
+  const gchar * root_path;
```
Comment (jaeyun-jung) [inline tests/capi/unittest_capi_service.cc:21]: Plz check coding style in this PR (unnecessary space between '*' and 'variable name'):
gchar * root_path > gchar *root_path

**Review by gichan-jang — APPROVED** [review]
LGTM

File: tests/capi/unittest_capi_service.cc (hunk 1 above)
Comment (again4you) [reply tests/capi/unittest_capi_service.cc:21]: After talking with @jaeyun-jung, I realized that gst-indent script is not suitable for C++ format.
I will submit the next version ASAP.

**Comment by again4you** [comment]
### Submit v2
* Update the code to comply with coding style.

**Review by jaeyun-jung — APPROVED** [review]

**Pull request merged**